routers/rooms.js

```diff
@@ -890,6 +916,134 @@ router.get('/room/:id/subrooms/:subroom_id/versions', authenticateToken, canView
+                $push: {
+                    "reports": {
+                        reason: reason,
+                        alleges_illegal_content: illegal_content,
+                        alleges_danger_of_harm: danger_of_harm,
+                        reporting_user_id: req.user.id,
+                        time: Date.now()
+                    }
+                }
+            }
+        );
+
+        const data = await collection.findOne(
+            {
+                _id: {
+                    $eq: id,
+                    $exists: true
+                }
+            }
+        );
+
+        if (illegal_content && danger_of_harm) {
+            auditLog(
+                `
+                !! EMERGENCY !!
+                <@&812976292634427394>
+                A player (ID ${req.user.id}) has submitted a report against room '${data.name}' (ID ${data._id})!
+                Reason: 
+                \`${reason}\`
+                HOWEVER
+                The user also indicated that this room may contain both ***ILLEGAL CONTENT*** and an ***IMMEDIATE THREAT TO HUMAN LIFE***!
+                It is absolutely paramount that this room is immediately investigated! Serious legal consequences may result if it is not!
+                Please remember it may be necessary to inform law enforcement of this incident, for that reason it is ***essential*** that
+                you keep a detailed log of your actions on this room and against this user. ***DO NOT DELETE ANY LOGS!***
+                `, true);
+        } else if (illegal_content) {
+            auditLog(
+                `
+                !! EMERGENCY !!
+                <@&812976292634427394>
+                A player (ID ${req.user.id}) has submitted a report against room '${data.name}' (ID ${data._id})!
+                Reason: 
+                \`${reason}\`
+                HOWEVER
+                The user also indicated that this room may contain ***ILLEGAL CONTENT!***
+                It is absolutely paramount that this room is immediately investigated! Serious legal consequences may result if it is not!
+                Please remember it may be necessary to inform law enforcement of this incident, for that reason it is ***essential*** that
+                you keep a detailed log of your actions on this room and against this user. ***DO NOT DELETE ANY LOGS!***
+                `, true);
+        } else if (danger_of_harm) {
+            auditLog(
+                `
+                !! EMERGENCY !!
+                <@&812976292634427394>
+                A player (ID ${req.user.id}) has submitted a report against room '${data.name}' (ID ${data._id})!
+                Reason: 
+                \`${reason}\`
+                HOWEVER
+                The user also indicated that this room may pose an ***IMMEDIATE THREAT TO HUMAN LIFE!***
+                It is absolutely paramount that this room is immediately investigated! Serious IRL consequences may result if it is not!
```

Comment:
Don't use acronyms like IRL

Reply:
this only gets shown to developers though

Reply:
Still, it's important to spell things out when dealing with potentially illegal content.

Reply:
see update